js/src/lib/components/InferenceWidget/widgets/FillMaskWidget/FillMaskWidget.svelte

```diff
@@ -142,7 +145,7 @@
 		<form>
 			{#if model.pipeline_tag === "fill-mask"}
 				<div class="mb-1.5 text-sm text-gray-500">
-					Mask token: <code>{model.mask_token}</code>
+					Mask token: <code>{model.mask_token ?? "unknown"}</code>
```

Comment:
on the hf-endpoints side, we sometimes don't have the mast token available right away.